src/ape/managers/config.py

```diff
@@ -37,17 +37,21 @@ class ConfigManager:
     PROJECT_FOLDER: Path
     name: str = ""
     version: str = ""
-    dependencies: List[str] = []
+    dependencies: Dict[str, str] = {}
     plugin_manager: PluginManager
-    _plugin_configs: Dict[str, ConfigItem] = dict()
 
-    def __post_init__(self):
-        config_file = self.PROJECT_FOLDER / CONFIG_FILE_NAME
+    _project_plugin_configs: Dict[str, Dict[str, ConfigItem]] = {}
+
+    @property
```

Comment:
In order to persist laziness throughout the New Year, we had to

1. Make this a "property" instead of happening at `__init__` time.
2. Make it cached per project. This way, if you change directories to another project (as the tests have to do), this can grab the new project's `ape-config.yaml` file changes. Yay!